packages/server/lib/browsers/electron.coffee

```diff
@@ -62,6 +71,12 @@ module.exports = {
           _win.on "close", ->
             if not child.isDestroyed()
               child.close()
+
+            ## give the child up to 1 second to close, then destroy it
+            setTimeout ->
+              if not child.isDestroyed()
+                child.destroy()
+            , 1000
```

Comment:
let's just always destroy the `child` window until we give users new API's to programmatically control whether or not they want the browser to accept the `beforeunload` dialog